api/src/main/java/org/apache/iceberg/catalog/Catalog.java

```diff
@@ -347,8 +347,10 @@ default boolean dropTable(TableIdentifier identifier) {
    * not cached, do nothing.
    *
    * @param identifier a table identifier
+   * @return true if the table is cached, false otherwise
```

Comment:
I think this is if the table "was in the cache" because "is" is misleading. It is no longer cached after this call.